npm/css-src/sass/_header.scss

```diff
@@ -1,4 +1,5 @@
 // Stylesheet also used in tdr-auth-server repo: https://github.com/nationalarchives/tdr-auth-server
+// Stylesheet also used in tdr-service-unavailable repo: https://github.com/nationalarchives/tdr-service-unavailable/blob/master/css-src/sass/_header.scss
 // Any changes should also be added to tdr-auth-server repo version
```

Comment:
Reword the comment to: "Any changes should also be added to the tdr-auth-server and tdr-service-unavailable repo versions."

Reply:
Good point. I've changed it.